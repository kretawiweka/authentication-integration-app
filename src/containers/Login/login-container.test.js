import React from 'react';
import { shallow } from 'enzyme';
import LoginContainer from './index';
import { Input } from 'antd';

describe('Login Container', () => {
  const wrapper = shallow(<LoginContainer />);
  it('should render my component', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('should render Input Component is 1', function() {
    expect(wrapper.find(Input).length).toBe(1);
  });
  it('should render Input.Password Component is 1', function() {
    expect(wrapper.find(Input.Password).length).toBe(1);
  });
});
