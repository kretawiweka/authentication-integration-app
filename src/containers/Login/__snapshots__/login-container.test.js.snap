// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Container should render my component 1`] = `ShallowWrapper {}`;
